product_form_pricelist/models/product_pricelist_item.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    applied_on = fields.Selection(
+        compute="_compute_applied_on_and_tmpl",
+        readonly=False,
+        store=True,
+    )
+    product_tmpl_id = fields.Many2one(
+        compute="_compute_applied_on_and_tmpl",
+        readonly=False,
+        store=True,
+    )
+
+    @api.depends("product_id", "product_tmpl_id")
+    def _compute_applied_on_and_tmpl(self):
```

Comment:
@sebastienbeau How does this behaviour fit standard code as additional values exist in Selection ???

https://github.com/odoo/odoo/blob/14.0/addons/product/models/product_pricelist.py#L385

Reply:
Ok, tested in interface, understood